Разметил главную страницу

Evidens2150 opened the pull request:
Разметил главную страницу, прошу проверить

Evidens2150 committed:
Разметил главную страницу

Разметил главную страницу

krxxl commented on Index.html:
```diff
@@ -0,0 +1,241 @@
+        <h1 class="poster-title">
+          THE OOH NOO <br />
+          COLLECTION
+        </h1>
```
здесь <br / > наверное не нужен, это сделается потом шириной элемента

krxxl commented on Index.html:
```diff
@@ -0,0 +1,241 @@
+        </ul>
+      </nav>
+      <ul class="header-list">
+        <li>
+          <button class="header-search-button">
+            <img src="search.png" alt="Search" />
+          </button>
+        </li>
+        <li>
+          <button class="header-contact-button button">
+            Contact Us
+          </button>
+        </li>
```
тут скорее все таки ссылки

Evidens2150 replied:
"Contact Us" сделал ссылкой, но мне кажется, при нажатии на Лупу должна появляться форма поиска.

krxxl commented on Index.html:
```diff
@@ -0,0 +1,241 @@
+        </h1>
+        <p>Designed in Slovenia</p>
+        <a class="button" href="#">EXPLORE NOW</a>
+        <p>FREE (CONTINENTAL) USA SHIPPINGON ALL ORDERS OF $100 USD OR MORE!</p>
```
посмотри на этот блок у него есть еще элементы оформи каким-нибудь списком кнопок

krxxl commented on Index.html:
```diff
@@ -0,0 +1,241 @@
+        </h2>
+        <ul class="category-list">
+          <li class="category-card">
+            <img
+              class="category-card-img"
+              src="apparel-kids.png"
+              alt="apparel kids"
+            />
+            <a class="category-card-name" href="#">APPAREL KIDS</a>
```
тут скорее всего весь блок должен быть ссылкой <a><img><p></p></a>

krxxl reviewed (changes requested):
есть несколько замечаний, в целом неплохо, я б везде убрал бы br

Evidens2150 committed:
Страница 1. Внёс исправления

Поправил ошибки, вставил "смысловые" изображения.

Evidens2150 committed:
Вторая страница

не совсем понял, как правильно отображать "путь до страницы" в первой секции на страницце.

krxxl reviewed (commented):
not bad
